js_modules/dagit/packages/core/src/schedules/SchedulesTable.tsx

```diff
@@ -177,7 +178,10 @@ const ScheduleRow: React.FC<{
       <td>
         {cronSchedule ? (
           <Tooltip position="bottom" content={cronSchedule}>
-            {humanCronString(cronSchedule)}
+            <span>
+              {humanCronString(cronSchedule)}
+              {executionTimezone ? ` ${timeZoneAbbr(executionTimezone)}` : ` UTC`}
```

Comment:
We also have `browserTimezoneAbbreviation`: https://github.com/dagster-io/dagster/blob/f53ff65cf7fb4b265d6fa68b8e944dacf9200706/js_modules/dagit/packages/core/src/app/time/browserTimezone.ts#L4

Reply:
Would you prefer I modify that to take in an optional timezone property and consume from there instead? I avoided it because its in `BrowserTimezone.ts` and this is deliberately not the browsers timezone (its the schedulers tz instead).

Reply:
🤦 Of course, you're right. We don't want the browser timezone. Never mind!

Reply:
Yea, its def'd and logged all the way over in scheduler.py, but I found it 

` timezone_str = external_schedule.execution_timezone
    if not timezone_str:
        timezone_str = "UTC"
        if log_verbose_checks:
            logger.warn(
                f"Using UTC as the timezone for {external_schedule.name} as it did not specify "
                "an execution_timezone in its definition."
            )`

Reply:
Oh interesting - good catch with the UTC fallback. I feel like in a perfect world we'd resolve that default value on the backend and the `executionTimezone` provided to us via the GraphQL interface wouldn't be nullable, but maybe we can look at changing that in a separate PR later!